tests/continuous/test_standard.py

```diff
@@ -774,7 +774,11 @@ def test_additive_bias(fcst, obs, reduce_dims, preserve_dims, weights, expected)
     result = scores.continuous.additive_bias(
         fcst, obs, reduce_dims=reduce_dims, preserve_dims=preserve_dims, weights=weights
     )
+    result2 = scores.continuous.mean_error(
```

Comment:
Can you make a note in the docstring for `test_additive_bias` that it also tests `mean_error`?

Reply:
done